apiserver/tools.go

```diff
@@ -91,12 +92,12 @@ func (h *toolsDownloadHandler) processGet(r *http.Request, st *state.State) ([]b
 		return nil, errors.Annotate(err, "error getting tools storage")
 	}
 	defer storage.Close()
-	_, reader, err := storage.Tools(version)
+	_, reader, err := storage.Open(v)
 	if errors.IsNotFound(err) {
-		// Tools could not be found in toolstorage,
+		// Tools could not be found in binarystorage,
```

Comment:
Again, probably "toolstorage" or "tool storage" is still appropriate here, I think.